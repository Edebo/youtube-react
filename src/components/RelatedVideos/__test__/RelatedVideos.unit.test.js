import React from "react";
import { shallow } from "enzyme";
import { RelatedVideos } from "../RelatedVideos";

describe("", () => {
  test("rendering the related video component", () => {
    const wrapper = shallow(<RelatedVideos />);
    expect(wrapper).toMatchSnapshot();
  });
});
