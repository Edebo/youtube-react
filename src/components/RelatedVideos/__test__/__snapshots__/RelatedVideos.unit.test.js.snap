// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering the related video component 1`] = `
<div
  className="related-videos"
>
  <NextUpVideo />
  <VideoPreview
    horizontal={true}
  />
  <VideoPreview
    horizontal={true}
  />
  <VideoPreview
    horizontal={true}
  />
</div>
`;
